python3-sys/src/lib.rs

```diff
@@ -48,6 +50,7 @@ pub use crate::pydebug::*;
 pub use crate::pyerrors::*;
 #[cfg(Py_3_4)]
 pub use crate::pyhash::*;
+pub use crate::pylifecycle::*;
```

Comment:
(detail) Since all the statements in `pylifecycle` are gated through `#[cfg(Py_3_8)]` wouldn't it be more consistent to just put that flag here? Same for `not(Py_LIMITED_API)` unless you think some of these might make their way to the limited API (although PEP 384 says it should be `pyconfig.h` and `Python.h` only)

Reply:
The reason I didn't gate all of `pylifecycle` is because there are symbols like `Py_Initialize` that are actually defined in `pylifecycle.h` but are currently defined elsewhere in `python3-sys`. I think the symbols were moved in Python 3.8. I have created the Rust `pylifecycle` module with the intent of eventually holding these symbols.

Reply:
found it ! Yes, that makes sense.